server/server.js

```diff
@@ -218,6 +219,30 @@ Server.prototype._handleResourceMessage = function(socket, message, messageType)
   }
 };
 
+// Process the resource messages associated with a single client
+Server.prototype._handleResourceMessages = function (socket, client) {
```

Comment:
This name is confusing (since there's a Server.prototype._handleResourceMessage). 

What do you think about renaming it to _restoreMessagesFromClient or _replayMessagesFromClient (I like this one, because it states better what it is doing).